lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+  attr_reader :items
+
+  ITEMS = {"Chips"         => 0.89,
+           "Chicken Tikka" => 3.99,
+           "Burger"        => 2.99,
+           "Pizza"         => 1.99,
+           "Wrap"          => 1.49
+          }
+  def initialize (items=ITEMS)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)